supply/lib/supply/setup.rb

```diff
@@ -44,7 +44,7 @@ def download_images(listing)
       IMAGES_TYPES.each do |image_type|
         if ['featureGraphic'].include?(image_type)
           # we don't get all files in full resolution :(
-          UI.message("Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphics. Please copy your feature graphics into `metadata/android/en-US/images/`")
```

Comment:
Do you know why it said "graphics" here? This makes me a bit sceptical if the change in error message is such a good idea...

Reply:
Looked into it.

Reply:
@janpio  Also, I shortened to 'graphic' because the Play Store only allows one graphic to be active.

![image](https://user-images.githubusercontent.com/19214773/34179203-ff9ffb64-e4d7-11e7-800d-0b46b953e78a.png)

Reply:
Ah, maybe because it is available in every language?